src/themes/default/views/feed.variables

```diff
@@ -190,6 +190,9 @@
 @invertedLikeHoverColor: @invertedSelectedTextColor;
 @invertedLikeActiveColor: @invertedLikeColor;
 
+@invertedMetadataActionColor: @invertedLightTextColor;
+@invertedMetadataActionHoverColor: @selectedTextColor;
```

Comment:
This color is too dark as it is declared in site.variables as `@selectedTextColor: rgba(0, 0, 0, 0.95);`
See https://jsfiddle.net/Festiis/sn82grqL/1/

Reply:
It should be `@invertedSelectedTextColor` instead which is `@invertedSelectedTextColor: rgba(255, 255, 255, 1);`
See https://jsfiddle.net/lubber/sdq5zbhv/

```suggestion
@invertedMetadataActionHoverColor: @invertedSelectedTextColor;
```